mobile/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -209,11 +209,10 @@
     <!-- Based on: https://github.com/PrivacyApps/donations/blob/747d36a18433c7e9329691054122a8ad337a62d2/Donations/src/main/res/values-zh/donations__strings.xml -->
     <string name="donations__button_close">關閉</string>
     <string name="donations__google_android_market">Google Play 商店</string>
-    <string name="donations__google_android_market_not_supported">不支援應用程式內購。Play 商店是否正確安裝?</string>
-    <string name="donations__google_android_market_donate_button">抖內！</string>
-    <string name="donations__google_android_market_text">抖內多少錢</string>
-    <string name="donations__thanks_dialog">感謝抖內！\n十分感謝您！</string>
-
-    <string name="donations__description">這個程式有用嗎？\n小額的抖內給開發人員用來支持此應用程式的開發</string>
+    <string name="donations__google_android_market_not_supported">不支援應用程式內捐助，Play 商店是否正確安裝？</string>
+    <string name="donations__google_android_market_donate_button">捐助！</string>
+    <string name="donations__google_android_market_text">捐助多少錢</string>
+    <string name="donations__thanks_dialog">感謝捐助！\n十分感謝您！</string>
```

Comment:
This part is an additional supplement to be more friendly to users. No need to update translations.